geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/process/unique/UniqueProcess.scala

```diff
@@ -267,6 +271,10 @@ class AttributeVisitor(val features: SimpleFeatureCollection,
     SelfClosingIterator(source.getFeatures(query).features()).foreach(addValue)
     uniqueValues.toMap
   }
+
+  override def getExpressions: java.util.List[Expression] ={
+    origSft.getAttributeDescriptors.map(ad => ff.property(ad.getLocalName)).toList
```

Comment:
this one should just need the attribute being queried on and any attributes in the filter I think